schemas/latest/devfile.json

```diff
@@ -9,7 +9,7 @@
     "attributes": {
       "description": "Map of implementation-dependant free-form YAML attributes.",
       "type": "object",
-      "additionalProperties": true
+      "additionalProperties": {}
```

Comment:
TODO: Take a look if it's a proper change or fix is needed in generator to revert it back